addons/point_of_sale/static/src/app/screens/scale_screen/scale_screen.js

```diff
@@ -94,4 +92,24 @@ export class ScaleScreen extends Component {
             this.state.tareLoading = false;
         }, 3000);
     }
+    get productPrice() {
```

Comment:
I find this method name not relevant anymore. Maybe more like a `get productUnitPrice` ? What do you think ?